dags/probe_scraper.py

```diff
@@ -91,3 +91,30 @@
         dag=dag)
 
     probe_expiry_alerts.set_upstream(probe_scraper)
+
+    delay_python_task = PythonOperator(
+        task_id="wait_for_30_minutes",
+        dag=dag,
+        python_callable=lambda: time.sleep(60 * 30))
+
+    lookml_generator = GKEPodOperator(
+        email=['frank@mozilla.com', 'dataops+alerts@mozilla.com'],
+        task_id='lookml_generator',
+        name='lookml-generator-1',
+        image='gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest',
+        gcp_conn_id="google_cloud_derived_datasets",
```

Comment:
```suggestion
        gcp_conn_id="google_cloud_airflow_gke",
        gke_cluster_name="workloads-prod-v1",
        gke_location="us-west1",
```
Use new gke cluster.